cmd/loadtest/account.go

```diff
@@ -670,7 +670,7 @@ func (ap *AccountPool) ReturnFunds(ctx context.Context) error {
 				Str("txHash", tx.Hash().Hex()).
 				Msg("transaction to return funds sent")
 
-			_, err = ap.waitMined(ctx, tx)
+			_, err = waitReceiptWithTimeout(ctx, ap.client, tx.Hash(), time.Minute)
```

Comment:
if the default timeout is one minute then we should use `waitReceipt` instead of `waitReceiptWithTimeout`

Reply:
Oh crap, you got me!

I did this to trick the linter and avoid this error:

```log 
cmd/loadtest/tx.go:25:6: func `waitReceiptWithTimeout` is unused (unused)
```

I want to keep the timeout variant method for future use. 

Would you mind us keeping this way in favor of having the `waitReceiptWithTimeout` implemented?